ci-deployer/Dockerfile

```diff
@@ -47,3 +47,9 @@ RUN curl -o /tmp/composer-setup.php https://getcomposer.org/installer \
 RUN php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} && rm -rf /tmp/composer-setup.php
 
 RUN composer global require deployer/deployer deployer/recipes wp-cli/wp-cli
+
+#Install phpcs with wordpress standards
+RUN composer global require squizlabs/php_codesniffer ~2.9.0 \
+  && mkdir /tmp/wpcs \
+  && curl -L https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/archive/master.tar.gz | tar xz --strip-components=1 -C /tmp/wpcs \
```

Comment:
We should fetch fresh standards every week so maybe add it in ci script directly?